cli/Cargo.toml

```diff
@@ -27,6 +27,7 @@ futures = "0.1.27"
 http = "0.1.17"
 hyper = "0.12.29"
 hyper-rustls = "0.16.1"
+indexmap = "1.0.2"
```

Comment:
Yep, thanks to `ImportMap` ordering of keys within the map is preserved - during parsing of map specifiers are sorted by length and alphabetically - resolution algorithm from spec iterates in that order. I thought it's better to do it once during parsing instead of sorting on each `ImportMap.resolve`